proposal/indented-syntax-improvements.md

```diff
@@ -0,0 +1,244 @@
+    .item-#{
+      $item
+    }
+      content: $item *
+        10
+````
+
+It will also be possible to include line breaks by wrapping the expression in
+parentheses.
+
+```sass
+@font-face
+  font-family: "Leatherman"
+  src: (
+    local("Leatherman"),
+    url("leatherman-COLRv1.otf") format("opentype") tech(color-COLRv1),
+    url("leatherman-outline.otf") format("opentype"),
+    url("leatherman-outline.woff") format("woff")
+  )
+```
+
+### Semicolons
+
+In the indented syntax outside of curly braces, authors can use a semicolon to
+explicitly end a statement. Subsequent lines in the same block still must have
+the same level of indentation. This means that the indented format won't support
+multiple statements on a single line, even if they are separated by a semicolon.
+
+```sass
+$font-stack: Helvetica, sans-serif;
+// or
+$primary-color: #333
+```
+
+### SCSS-in-Sass
+
+Alternatively, authors can use SCSS block syntax with braces and semicolons
+within an indented document.
+
+```sass
+a
+  color: blue
+
+.grid {
+  display: grid;
+  grid-template:
+    'logo title copy' auto
+    'alert alert alert' minmax(0, auto) / 1fr 1fr 1fr;
+}
+```
+
+The braces essentially let authors opt in to SCSS format for part of the
+document. This means that authors must use semicolons to separate statements
+inside that block. Indentation rules do not apply and multiple statements can be
+on a single line, separated by semicolons.
+
+```sass
+$font-stack: Helvetica, sans-serif
+$primary-color: #333
+body {
+  font: 100% $font-stack;
+  color: $primary-color; background: blue;
+}
+```
+
+### Ending statements
+
+When line breaks do not end statements, the line breaks are treated as white
+space, and do not have any requirements about the amount of indentation. The
+next statement's indentation is compared to the indentation at the start of the
+current statement to determine nesting or block closure.
+
+By design, line breaks are ignored as meaningless white space except in
+contexts where the semantics define that a statement may end. When an author
+inserts a line break they intend to be meaningless in a position where a
+statement may end, the statement will end, and parsing will most likely fail on
+the subsequent text. The following places are where line breaks will cause
+statements to end.
+
+#### After a SassScript value
+
+In a simple declaration, `$foo: bar \n`, the line break must cause the statement
+to end.
+
+This may be surprising in more complex situations, for instance, with binary
+operators. `$foo: 3\n+ 4\n` ends the statement after `3`, but `$foo: 3 +\n4\n`
+ends the statement after `4`. Wrapping with parentheses allows authors more
+flexibility with `$foo: (3\n+ 4)`.
+
+This also applies to flow control at-rules. `@if $a \n and $b` would end the
+statement after `$a`, but `@if ($a \n and $b)` can be parsed.
+
+#### After a non-enclosed list begins
+
+A line break in a list that is not in a `BracketedListExpression` or enclosed in
```

Comment:
Since this is still the summary, we shouldn't assume that readers will have familiarity with spec production names. I'd just say "not enclosed in `[]` or `()`".